connector_salesforce/README.rst

```diff
@@ -0,0 +1,247 @@
+Salesforce Connector
+====================
+
+This module allows to import and export various models between
+Odoo and Salesforce.
+
+It provides the foundations to create or extend various synchronizations.
+
+By default following synchronisations are supported out of the box:
+
+ - Import of Account and contact into Odoo Partners. Salesforce is the master.
+   Parent relation of Sales force account is not supported at this time.
+ - Import or export of Product. ERP or Salesforce can be master. Bidirectional synchronisation is not supported and a master must be set.
```

Comment:
- **Product synchronization**. ERP or Salesforce can be master but bidirectional synchronization is not supported so a master must be set.